parent: !include default.yaml
pr2_a:
  parent: !include robots/pr2.yaml
pr2_b:
  parent: !include robots/pr2.yaml
robot_interface:
  /pr2_a/base:
    plugin: SendFollowJointTrajectory
    namespace: /pr2_a/whole_body_controller/base/follow_joint_trajectory
    state_topic: /pr2_a/whole_body_controller/base/state
    fill_velocity_values: True
    robot_name: pr2_a
  /pr2_a/body:
    plugin: SendFollowJointTrajectory
    namespace: /pr2_a/whole_body_controller/body/follow_joint_trajectory
    state_topic: /pr2_a/whole_body_controller/body/state
    fill_velocity_values: False
    robot_name: pr2_a
  /pr2_b/base:
    plugin: SendFollowJointTrajectory
    namespace: /pr2_b/whole_body_controller/base/follow_joint_trajectory
    state_topic: /pr2_b/whole_body_controller/base/state
    fill_velocity_values: True
    robot_name: pr2_b
  /pr2_b/body:
    plugin: SendFollowJointTrajectory
    namespace: /pr2_b/whole_body_controller/body/follow_joint_trajectory
    state_topic: /pr2_b/whole_body_controller/body/state
    fill_velocity_values: False
    robot_name: pr2_b
